packages/core/src/configuration.ts

```diff
@@ -126,6 +129,9 @@ function getEndpoint(type: string, conf: TransportConfiguration, source?: string
   const tld = conf.datacenter === 'us' ? 'com' : 'eu'
   const domain = conf.env === 'production' ? `datadoghq.${tld}` : `datad0g.${tld}`
   const tags = `version:${conf.version}`
-  return `https://${type}-http-intake.logs.${domain}/v1/input/${conf.clientToken}?ddsource=${source ||
-    'browser'}&ddtags=${tags}`
+  const datadogHost = `${type}-http-intake.logs.${domain}`
+  const host = conf.proxyHost ? conf.proxyHost : datadogHost
+  const proxyParameter = conf.proxyHost ? `dd-host=${datadogHost}&` : ''
```

Comment:
All the other query params don't contain an hyphen like `dd-host`.
We should maybe rename it to `ddhost`?